apps/mobile/src/screens/Login/OnboardingEmailScreen.tsx

```diff
@@ -69,9 +88,41 @@ export function OnboardingEmailScreen() {
       }
     }
 
+    if (!EMAIL_FORMAT.test(email)) {
```

Comment:
can we just use `if (isInvalidEmail)` ?

Reply:
Good catch! Updated to use `isInvalidEmail`